README.md

```diff
@@ -34,7 +34,8 @@ If you use this software in an academic work, please cite:
 
 ## Installation notes
 
-DBoW2 requires [DLib](https://github.com/dorian3d/DLib), which is automatically installed if it cannot be found in the system. You can also find it in [my repository](https://github.com/dorian3d/DLib).
+ 
```

Comment:
I think you can remove the line and does not add any comment. Since DLib is not longer needed, there is no mention to make.